sample-skill/lambda/alexa/settings.py

```diff
@@ -0,0 +1,13 @@
+"""Settings for the Alexa skill."""
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+API_DOMAIN = os.getenv("API_DOMAIN")
```

Comment:
is this for eox-core? what do you think about EOX_CORE_API_DOMAIN? So it's more explicit.

Reply:
This is for edX and `eox-core` API, maybe a better name could be `LMS_DOMAIN`? What do you think?

Reply:
LGTM!